source/common/http/http2/codec_impl.cc

```diff
@@ -1065,6 +1075,11 @@ ConnectionImpl::ClientHttp2Options::ClientHttp2Options(const Http2Settings& http
                                                  Http2Settings::DEFAULT_MAX_CONCURRENT_STREAMS);
 }
 
+void ConnectionImpl::onError(int lib_error_code, const char* msg, size_t len) {
+  ENVOY_CONN_LOG(trace, "nghttp2 error: lib_error_code: {}, message: {}", connection_,
```

Comment:
I imagine this won't be called very often, any chance we could get it bumped to debug so we don't have to turn on _as much_ of a firehose to see this?

Reply:
Good point. I bumped to 'debug'.